extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/JsonProviderProvider.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.smallrye.graphql.runtime;
+
+import jakarta.json.spi.JsonProvider;
+
+public class JsonProviderProvider {
```

Comment:
Would `JsonProviderHolder` sound less silly? :)

Reply:
+1 for the name change but I also think this class should be moved to the `quarkus-jsonp` extension so that we can share this globally.

It should be put in the runtime module in the `io.quarkus.jsonp` package (as it's more of an API).